EXILED/Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs

```diff
@@ -89,17 +89,18 @@ public static bool CheckFriendlyFirePlayerRules(Footprint attackerFootprint, Ref
 
             // Return false, no custom friendly fire allowed, default to NW logic for FF. No point in processing if FF is enabled across the board.
             if (Server.FriendlyFire)
-                return HitboxIdentity.IsEnemy(attackerFootprint.Role, victimHub.roleManager.CurrentRole.RoleTypeId);
+                return HitboxIdentity.IsDamageable(attackerFootprint.Hub, victimHub);
 
-            // always allow damage from Server.Host
+            // Always allow damage from Server.Host
             if (attackerFootprint.Hub == Server.Host.ReferenceHub)
                 return true;
 
             // Only check friendlyFire if the FootPrint hasn't changed (Fix for Grenade not dealing damage because it's from a dead player)
             // TODO rework FriendlyFireRule to make it compatible with Footprint
-            if (!attackerFootprint.SameLife(new(attackerFootprint.Hub)))
-                return HitboxIdentity.IsEnemy(attackerFootprint.Role, victimHub.roleManager.CurrentRole.RoleTypeId);
+            if (!attackerFootprint.SameLife(new Footprint(attackerFootprint.Hub)))
+                return HitboxIdentity.IsDamageable(attackerFootprint.Hub, victimHub);
```

Comment:
This return need to check with footprint value not player

Reply:
See previous comments.